src/Adapter/AdapterMessageConverter.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/*
+ * This file is part of the prooph/event-store.
+ * (c) 2014-2015 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ * 
+ * Date: 8/14/15 - 8:06 PM
+ */
+
+namespace Prooph\EventStore\Adapter;
+
+use Prooph\Common\Messaging\Message;
+use Prooph\Common\Messaging\MessageConverter;
+use Prooph\Common\Messaging\MessageFactory;
+use Prooph\EventStore\Adapter\Exception\ConfigurationException;
+use Prooph\EventStore\Adapter\Exception\RuntimeException;
+
+/**
+ * Trait AdapterMessageConverter
+ *
+ * This trait adds message conversion capabilities to any event store adapter.
+ *
+ * @package Prooph\EventStore\Adapter
+ * @author Alexander Miertsch <alexander.miertsch.extern@sixt.com>
+ */
+trait AdapterMessageConverter
```

Comment:
yea I first named it this way but I stopped putting Interface behind each interface. To be consistent with naming I think it is the same for traits. IMO the name should only represent the main purpose of the class, trait, interface, whatever. That it is a trait is already covered by the definition.